code/__DEFINES/mobs.dm

```diff
@@ -67,9 +67,7 @@
 #define LARVA_BODYPART "larva"
 /*see __DEFINES/inventory.dm for bodypart bitflag defines*/
 
-// Health/damage defines
-#define MAX_LIVING_HEALTH 100
```

Comment:
Why remove this?

Reply:
This wasn't intentional, I actually atomized this change and it seems git auto-resolve of conflicts pulled a prank on me.
Fixing it.